### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by glenrobson** [IIIF-Commons/thumbnail-panel#48]
Need accessibility statement 

https://github.com/UniversalViewer/universalviewer/wiki/Accessibility-Statement-for-the-Universal-Viewer

Keyboard navigation: tab into panel, left / right arrow keys  to view images, tab to leave panel. Could also use this:

https://www.w3.org/WAI/ARIA/apg/patterns/listbox/

We can also look at testing with lighthouse: https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk

Can add unit tests to ensure keyboard navigation remains accessible.

Could write a single accessibility statement for iiif-commons that all components will meet.

**Comment by glenrobson** [comment]
We should be careful about stating accessibility level as this would be integrated into another product which would have to evaluate the whole app rather than individual components.

**Comment by stephenwf** [comment]
Yes, we can't promise an accessibility standard for a few reasons:

- CSS is provided from the outside
- Some customisations may change how accessible the panel is
- The IIIF is unknown, and without good labelling in the content - some of the accessibility requirements for describing image content will be impossible.

But we can make the interactions themselves accessible, using the W3C provided patterns.
